src/main/java/org/openpnp/model/Part.java

```diff
@@ -50,7 +50,6 @@
     @Attribute(required = false)
     private int pickRetryCount = 0;
 
-
```

Comment:
Artifact.

Reply:
Fixed in b7dd9a39d9c27824bea9288f75a8d8aaeb0ca33d